code/modules/surgery/organs/vocal_cords.dm

```diff
@@ -260,31 +257,12 @@
 	var/static/regex/multispin_words = regex("like a record baby|right round")
 
 	var/i = 0
-	//STUN
-	if(findtext(message, stun_words))
-		cooldown = COOLDOWN_STUN
-		for(var/V in listeners)
-			var/mob/living/L = V
-			L.Stun(60 * power_multiplier)
-
-	//KNOCKDOWN
-	else if(findtext(message, knockdown_words))
-		cooldown = COOLDOWN_STUN
-		for(var/V in listeners)
-			var/mob/living/L = V
-			L.Paralyze(60 * power_multiplier)
```

Comment:
this could probably just be made into an actual knockdown instead of removed for being hard CC